lms/djangoapps/course_api/blocks/tests/test_views.py

```diff
@@ -232,6 +245,18 @@ def test_with_list_field_url(self):
         )
         self.verify_response_with_requested_fields(response)
 
+    def test_discussion_student_view_data(self):
```

Comment:
This test should be moved to the tests for the discussion module.
The Course Blocks API is generic for all blocks.  We don't want to clutter its tests by verifying `student_view_data` for all block types.